### PR TITLE
Network Error Logging: frontend part

### PR DESCRIPTION
**Pull request by oioki**
Frontend part of the NEL ([Network Error Logging](https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging)) implementation.

Example:
image

Related PRs:
https://github.com/getsentry/sentry/pull/55135
https://github.com/getsentry/relay/pull/2421

### CODE REVIEW

File: static/app/components/events/interfaces/nel/help/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import ExternalLink from 'sentry/components/links/externalLink';
+import {IconOpen} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+
+import nelProperties from './nelProperties';
+
+function NELHelp({data}) {
+  const getLink = () => {
+    const href =
+      'https://developer.mozilla.org/en-US/docs/Web/HTTP/Network_Error_Logging#error_reports';
```
Comment (ArthurKnaus) [inline static/app/components/events/interfaces/nel/help/index.tsx:12]: Move that constant outside the component.

File: static/app/components/events/interfaces/nel/help/index.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    return (
+      <StyledExternalLink href={href}>
+        {'developer.mozilla.org'}
+        <IconOpen size="xs" className="external-icon" />
+      </StyledExternalLink>
```
Comment (ArthurKnaus) [inline static/app/components/events/interfaces/nel/help/index.tsx:18]: nit: Why does it need to be in a separate function. IMHO this simply could be inlined in the main JSX.

**Review by ArthurKnaus — COMMENTED** [review]
Awesome feature! 🚀 
Got a few remarks:

File: static/app/components/events/interfaces/nel/help/nelProperties.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  server_ip: t(
+    `The IP address of the server to which the user agent sent the request, if available. Otherwise, an empty string.`
+  ),
+  protocol: t(
+    `The network protocol used to fetch the resource as identified by the ALPN Protocol ID, if available. Otherwise, "".`
+  ),
+  method: t(`request's request method.`),
+  request_headers: t(
+    `The result of executing <a href="https://w3c.github.io/network-error-logging/#extract-request-headers" target="_blank" rel="noreferrer">5.2 Extract request headers</a> on <em>request</em> and <em>policy</em>.`
+  ),
```
[the remark this answers is not in the document]
Comment (scttcper) [reply static/app/components/events/interfaces/nel/help/nelProperties.tsx:17]: they can't be template tags or contain html

**Pull request closed without merging**